AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -82,34 +82,22 @@ public async Task<ActionResult<HearingDetailsResponse>> Post([FromBody] BookHear
                     var endpointsWithDa = newBookingRequest.Endpoints
                         .Where(x => !string.IsNullOrWhiteSpace(x.DefenceAdvocateContactEmail))
                         .ToList();
-                    _hearingsService.AssignEndpointDefenceAdvocates(endpointsWithDa,
-                        newBookingRequest.Participants.AsReadOnly());
+                    _hearingsService.AssignEndpointDefenceAdvocates(endpointsWithDa, newBookingRequest.Participants.AsReadOnly());
                 }
 
                 newBookingRequest.CreatedBy = _userIdentity.GetUserIdentityName();
                 _logger.LogInformation("BookNewHearing - Attempting to send booking request to Booking API");
                 var hearingDetailsResponse = await _bookingsApiClient.BookNewHearingAsync(newBookingRequest);
-                _logger.LogInformation("BookNewHearing - Successfully booked hearing {Hearing}",
-                    hearingDetailsResponse.Id);
+                _logger.LogInformation("BookNewHearing - Successfully booked hearing {Hearing}", hearingDetailsResponse.Id);
 
-                if(hearingDetailsResponse.Status == BookingStatus.Failed)
-                {
-                    return Created("", hearingDetailsResponse);
-                }
-                var judgeExists = request.BookingDetails.Participants?.Any(x => x.HearingRoleName == RoleNames.Judge) ?? false;
-                if (judgeExists)
-                {
-                    await GetConferenceStatus(hearingDetailsResponse.Id, 
-                        $"Failed to get the conference from video api, possibly the conference was not created or the kinly meeting room is null - hearingId: {hearingDetailsResponse.Id}");
-                }
```

Comment:
@marcogagliardi Have just realised, this condition still needs to be on to decided whether we check the conference status, should've noticed it when I did my refactor -- when a booking is made without a judge, it isnt confirmed so we need to maybe add this condition to the client side. If booking doesnt contain a judge -- dont check the conference status

Reply:
done